.mergify.yml

```diff
@@ -207,6 +207,19 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 8.4 branch
```

Comment:
The same block already exists below at line 223, likely because this shouldn't run for patch releases, only major/minor releases that create a new branch.

Reply:
Yah, assuming this can be closed. Tell me and I'll re-open it if I'm wrong, hah.